frontend/src/store/editorSlice.ts

```diff
@@ -1,27 +1,39 @@
-import { createSlice } from "@reduxjs/toolkit";
 import type { PayloadAction } from "@reduxjs/toolkit";
-import { RootState } from "./store";
+import { createSlice } from "@reduxjs/toolkit";
+import { EditorView } from "codemirror";
 import * as yorkie from "yorkie-js-sdk";
-import { YorkieCodeMirrorDocType, YorkieCodeMirrorPresenceType } from "../utils/yorkie/yorkieSync";
 import { ShareRole } from "../utils/share";
-import { EditorView } from "codemirror";
+import { YorkieCodeMirrorDocType, YorkieCodeMirrorPresenceType } from "../utils/yorkie/yorkieSync";
+import { RootState } from "./store";
+
+export enum EditorModeType {
+	edit = "edit",
+	both = "both",
+	read = "read",
+}
+
+export enum CodeKeyType {
+	sublime = "sublime",
+	vim = "vim",
+}
 
-export type EditorModeType = "edit" | "both" | "read";
 export type CodePairDocType = yorkie.Document<
 	YorkieCodeMirrorDocType,
 	YorkieCodeMirrorPresenceType
 >;
 
 export interface EditorState {
 	mode: EditorModeType;
+	codeKey: CodeKeyType;
```

Comment:
How about move this options to `configSlice`?
In `configSlice`, it is saved in persistence storage (local storage).

Reply:
I got it! And I think it would be good to provide a clear explanation of the `Slice` in frontend code.